Installer/kan-test-installer.sh

```diff
@@ -363,71 +363,86 @@ while [ $current_step -lt 6 ]; do
 
     ;;
     5 )
+        echo "Agree to collect telemetry? (y/n)"
```

Comment:
Change the text to “May we collect anonymous usage data to help improve the app's performance and user experience?"